### PR TITLE
Enze/reorg

### PR DESCRIPTION
**Pull request by enze-chen**
Opening this PR to test CI. Still a WIP.

### CODE REVIEW

**Comment by enze-chen** [comment]
Reorg and tweaks are done. Just waiting on Travis-CI and CitrinationClient.

File: .travis.yml [hunk 1]
```diff
@@ -15,22 +15,27 @@ install:
...
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/ImportInstron"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/TsneDemo"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/clients_sequence/1_data_client_api_tutorial"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/clients_sequence/2_data_views_client_api_tutorial"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/clients_sequence/3_models_client_api_tutorial"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/clients_sequence/4_search_client_api_tutorial"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/1_ImportVASP"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/2_WorkingWithPIFs"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/3_IntroQueries"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/4_MLonCitrination"
```
Comment (maxhutch) [inline .travis.yml:36]: What about `5_sequential_learning_api_tutorial.ipynb`?

File: .travis.yml [hunk 2]
```diff
@@ -15,22 +15,27 @@ install:
...
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/AdvancedPif"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_api_examples/tutorial_sequence/AdvancedQueries"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "citrination_ui_examples/JournalPaperToModel"
+- ./ci/test_notebooks.sh $NOTEBOOK_KERNEL "synthetic_data_examples/SyntheticDataDemo"
```
Comment (maxhutch) [inline .travis.yml:40]: What about `BandGapComparisonCitrinationVsMP.ipynb` and `DataRetrievalWithMatMiner.ipynb`?

**Review by maxhutch — COMMENTED** [review]
Three of the notebooks don't seem to be tested in `.travis.yml`.  Can those be added, or is there something about them that keeps us from automatically testing them?

The new repo organization and readmes look very nice.  I didn't go through each notebook, though.  If there are some that have significant changes, let me know and I can look at them specifically.  Its really hard to tell from just the diff.

**Comment by enze-chen** [comment]
Hi @maxhutch, 

First, thank you for the review, and the feedback. Absolutely I don't want you reading through every notebook; I think they're OK because they've been independently checked in the past.

To address your comments about the other notebooks:

1. The `sequential_learning` tutorial I left out of the check because it would take the longest to run. It should pass all tests just fine, it'll just take a while. Thoughts?

1. The Matminer examples will require `matminer` and `pymatgen` to be included in the `requirements.txt` file. Which, I guess should be in there if we even have these notebooks in this repo to begin with, and hopefully will have no installation problems. I just didn't include them because they originally weren't included. Thoughts on adding these?

I know your comment says that you would like them included for completeness. :smile: I just want to confirm, haha.

**Comment by maxhutch** [comment]
> 1. The `sequential_learning` tutorial I left out of the check because it would take the longest to run. It should pass all tests just fine, it'll just take a while. Thoughts?

How long is long?

> 2. The Matminer examples will require `matminer` and `pymatgen` to be included in the `requirements.txt` file. Which, I guess should be in there if we even have these notebooks in this repo to begin with, and hopefully will have no installation problems. I just didn't include them because they originally weren't included. Thoughts on adding these?

Yeah, I think we should just add them.  If we were worried about requiring too much, we could move `requirements.txt` into the sub-directories and then thin them to only the dependencies in that sub-directory.  I think that's a premature optimization, though; let's wait to hear someone complain about needing to install too much.

**Comment by enze-chen** [comment]
Good call on both. Not long in absolute time (I estimate 10 minutes?), just relatively (particularly when I was just trying to get CI to work). I'll add both in today/tomorrow.

**Comment by maxhutch** [comment]
10 minutes is long for travis; I'd go ahead and skip it (maybe make a comment in the `.travis.yml` file or `ci/README.md` file.

**Review by maxhutch — APPROVED** [review]
Looks great!

**Pull request merged**